actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -619,12 +620,8 @@ def apply_form_for_options!(object, options) # :nodoc:
       #   When set to <tt>true</tt>, the form is submitted via standard HTTP.
       #   When set to <tt>false</tt>, the form is submitted as a "remote form", which
       #   is handled by Rails UJS as an XHR. When unspecified, the behavior is derived
-      #   from <tt>config.action_view.form_with_generates_remote_forms</tt> where the
-      #   config's value is actually the inverse of what <tt>local</tt>'s value would be.
-      #   As of Rails 6.1, that configuration option defaults to <tt>false</tt>
-      #   (which has the equivalent effect of passing <tt>local: true</tt>).
-      #   In previous versions of Rails, that configuration option defaults to
-      #   <tt>true</tt> (the equivalent of passing <tt>local: false</tt>).
+      #   from <tt>config.action_view.form_with_generates_local_forms</tt> where the
+      #   default is <tt>false</tt> (the equivalent of passing <tt>local: false</tt>).
```

Comment:
```suggestion
      #   default is <tt>false</tt>.
```

I don't think you need to say this anymore, since the settings make sense now.